lib/atlassian-jwt-authentication/verify.rb

```diff
@@ -66,7 +66,11 @@ def verify
       # Decode the token again, this time with signature & claims verification
       options = JWT::DefaultOptions::DEFAULT_OPTIONS.merge(verify_expiration: AtlassianJwtAuthentication.verify_jwt_expiration).merge(decode_options)
       decoder = JWT::Decode.new(jwt, decode_key, true, options)
-      payload, header = decoder.decode_segments
+      begin
+        payload, header = decoder.decode_segments
+      rescue JWT::VerificationError, JWT::ExpiredSignature
+        return false
```

Comment:
See https://github.com/MeisterLabs/atlassian-jwt-authentication/blob/3fc48fe9c529d7ed4adb0f15405d57c1df108dbb/lib/atlassian-jwt-authentication/verify.rb#L27 

When trying to decode JWT for the first time we log an error message if it fails.

I think we should do the same here so we are consistent and are tracking it (if logger is enabled)